src/world_interface/real_world_constants.h

```diff
@@ -29,6 +29,39 @@ constexpr auto pidMotors = frozen::make_unordered_set<motorid_t>(
 	{motorid_t::armBase, motorid_t::shoulder, motorid_t::elbow, motorid_t::forearm,
 	 motorid_t::wrist});
 
+// struct PotenetiometerParams => potparams_t
+/**
+ * @brief Represents parameters defining a potentiometer scale.
+ *
+ * Contains two joint angles in millidegrees and their associated potentiometer ADC values;
+ * this defines a linear scale from potentiometer ADC value to joint angle that can be sent to
+ * the motor boards for position control and feedback.
+ */
+struct potparams_t {
+	/** The "low" point on the ADC scale. */
+	uint16_t adc_lo;
+	/** The "low" point on the joint rotation scale. */
+	int32_t mdeg_lo;
+	/** The "high" point on the ADC scale. */
+	uint16_t adc_hi;
+	/** The "high" point on the joint rotation scale. */
+	int32_t mdeg_hi;
+};
+
+struct encparams_t {
+	bool isInverted;
+	int pulses_per_joint_revolution;
+};
+
+constexpr auto encMotors = frozen::make_unordered_map<motorid_t, encparams_t>(
+	{{motorid_t::shoulder, {.isInverted = false, .pulses_per_joint_revolution = 20 * 1000}},
+	 {motorid_t::elbow, {.isInverted = true, .pulses_per_joint_revolution = 36 * 1000}}});
```

Comment:
You have the numbers to calculate the ppjrs, can you put them here instead of these dummy values?